src/twisted/mail/newsfragments/8487.bugfix

```diff
@@ -0,0 +1 @@
+twistd.mail.scripts.mailmail has been ported to Python 3
```

Comment:
Is this a bug? I would say that is a feature :) Don't forget the full stop at the end to be explicit that the sentence ends there and that it was not accidentally left unfinished.

Reply:
Well I could argue that since no new functionality is being added by this PR,
the fact that mailmail does not work on Python 3 is a bug, so this PR is a bugfix.